config/thoth.yaml

```diff
@@ -141,22 +141,22 @@ repositories:
           labels: [bot]
           changelog_file: true
 
-  - slug: thoth-station/package-releases-job
-    token: '{KEBECHET_TOKEN}'
-    managers:
-      - name: update
-        configuration:
-          labels: [bot]
-      - name: info
-      - name: version
-        configuration:
-          maintainers:
-            - goern
-            - fridex
-          assignees:
-            - sesheta
-          labels: [bot]
-          changelog_file: true
+  - slug: thoth-station/package-releathamosses-job
+    token: '{KEBECHET_TOKEN}'thamos
+    managers:thamos
+      - name: updatethamos
+        configuration:thamos
+          labels: [bot]thamos
+      - name: infothamos
+      - name: versionthamos
+        configuration:thamos
+          maintainers:thamos
+            - goernthamos
+            - fridexthamos
+          assignees:thamos
+            - seshetathamos
+          labels: [bot]thamos
+          changelog_file: truethamos
```

Comment:
This is somehow weird :-/

Reply:
uh... :)
fixed in https://github.com/thoth-station/kebechet/commit/be53e2d9c237609e3c5c76bcc3202c257558853b